test/t8_schemes/t8_gtest_root.cxx

```diff
@@ -0,0 +1,61 @@
+/*
+This file is part of t8code.
+t8code is a C library to manage a collection (a forest) of multiple
+connected adaptive space-trees of general element classes in parallel.
+
+Copyright (C) 2015 the developers
```

Comment:
```suggestion
Copyright (C) 2024 the developers
```
Using 'Configure User Snippets` in VsCode you can implement your own snippet to fill in the license with the current year ;)